src/index.js

```diff
@@ -227,30 +227,20 @@ export class ULabel {
                 case ul.config.decrease_brush_size_keybind:
                     ul.change_brush_size(1/1.1);
                     break;
+                case ul.config.change_zoom_keybind.toLowerCase():
+                    ul.show_initial_crop();
+                    break;
+                case ul.config.change_zoom_keybind.toUpperCase():
+                    ul.show_whole_image();
+                    break;
                 default:
                     if (!DELETE_MODES.includes(current_subtask.state.spatial_type)) {
                         // Check for class keybinds
                         for (let i = 0; i < current_subtask.class_defs.length; i++) {
                             const class_def = current_subtask.class_defs[i];
                             if (class_def.keybind !== null && e.key === class_def.keybind) {
-                                // Get the active or hovered annotation id
-                                let target_id = null;
-                                if (current_subtask.state.active_id !== null) {
-                                    target_id = JSON.parse(JSON.stringify(current_subtask.state.active_id));
-                                } else if (current_subtask.state.move_candidate !== null) {
-                                    target_id = JSON.parse(JSON.stringify(current_subtask.state.move_candidate["annid"]));
-                                }
-
-                                if (target_id === null) {
-                                    // Set the active class to the class with the matching keybind
-                                    ul.update_id_toolbox_display(i);
-                                    // Recolor the brush circle
-                                    ul.recolor_brush_circle();
-                                } else {
-                                    // Set the annotation's class to the class with the matching keybind
-                                    ul.handle_id_dialog_click(ul.state["last_move"], target_id, i);
-                                }
-                                                            
+                                // Click the class button
+                                $(`#tb-id-app--${ul.state["current_subtask"]} a.tbid-opt`).eq(i).click();                            
```

Comment:
```suggestion
                                $(`#tb-id-app--${ul.state["current_subtask"]} a.tbid-opt`).eq(i).trigger("click");                            
```
`.click` is deprecated. The jQuery documentation suggests using `.trigger("click")` instead.